From 2a5f85c95dad20ddd1ee4c85bd85f7c30e48cf68 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Tue, 7 Sep 2010 13:23:28 +0000
Subject: [PATCH v2 06/20] Revert "gettextize: git-revert mark the "me" variable for translation"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The `me' variable can be either "revert" or "cherry-pick". Some of
these should be translated, but it's used in too many different
contexts for the approach I initially took to be useful.

Reported-by: Jonathan Nieder <jrnieder@gmail.com>
Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 builtin/revert.c |    4 +---
 1 files changed, 1 insertions(+), 3 deletions(-)

diff --git a/builtin/revert.c b/builtin/revert.c
index 7157ee1..a93a5ba 100644
--- a/builtin/revert.c
+++ b/builtin/revert.c
@@ -553,9 +553,7 @@ static int revert_or_cherry_pick(int argc, const char **argv)
 	struct rev_info revs;
 
 	git_config(git_default_config, NULL);
-	/* TRANSLATORS: This is used in several error messages indicating
-	   the name of the current program */
-	me = action == REVERT ? N_("revert") : N_("cherry-pick");
+	me = action == REVERT ? "revert" : "cherry-pick";
 	setenv(GIT_REFLOG_ACTION, me, 0);
 	parse_args(argc, argv);
 
-- 
1.7.2.3.313.gcd15

